src/views/AboutUs.vue

```diff
@@ -100,7 +146,59 @@ export default {
         subheading:
           "Self Learning sits at the heart of devlup labs, we believe in proper utilization of resources and peer guidance to propel self learning."
       }
-    ]
+    ],
+    valid: false,
+    links: [
```

Comment:
Remove the these lines as links aren't being used now

Reply:
the changes have been applied